docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -17,7 +17,7 @@ The main features of the `Tizen.Multimedia.Camera` class include:
 
     You can preview images in real time with the `StartPreview()` method of the `Tizen.Multimedia.Camera` class. The feature provides:
```

Comment:
How about changing this paragraph as follow?

---
You can preview images in real time with the `StartPreview()` method of the `Tizen.Multimedia.Camera` class. 
Camera provides support for the following features:
 
-   Pixel formats, such as NV12, NV12T, NV16, NV21, YUYV, UYVY, YUV420P, I420, Yv12, Rgb565, Rgb888, Rgba, Argb, Jpeg, H264 and Invz
-  Preview...
- Rotation...

Reply:
I updated statements.